CodeBeam.MudExtensions/Components/Stepper/MudStepper.razor

```diff
@@ -51,50 +51,60 @@
             </div>
             
             <div class="d-flex gap-4">
+                
+                @{
+                    bool showResultStep = ShowResultStep();
+                }
+
+                @if (!DisablePreviousButton && ActiveIndex != 0)
+                {
+                    if (ActiveIndex < Steps.Count && Steps[ActiveIndex].Status != StepStatus.Continued || showResultStep)
+                    {
+                        <MudIconButton Color="@Color" Variant="@Variant" Icon="@Icons.Filled.ChevronLeft" OnClick="@(() => SetActiveIndex(-1))" />
+                    }
+                    else
+                    {
+                        <MudButton Color="@Color" Variant="@Variant" OnClick="@(() => SetActiveIndex(-1))">@LocalizedStrings.Previous</MudButton>
+                    }
+                }
+
                 @if (ActionContent != null)
                 {
                     @ActionContent
                 }
-                else
+
+                <MudSpacer />
```

Comment:
If i understand true, the action content placed after "Previous" button. There is no probability to place a button near to "Next" button due to this MudSpacer.

I think users should disable/remove this MudSpacer also to have maximum flexibility.

Reply:
Maybe like
```
@if (ActionContent != null)
{
    @ActionContent
}
else
{
    <MudSpacer />
}
```
So users can place the MudSpacer at the start or at the end to control the inline-button's placement. Isn't it better?

Reply:
I've updated as suggested and added a remark to `ActionContent` about this behavior